### PR TITLE
Changes Dockerfile for faster iteration builds

### LINKED ISSUE #427: Change Dockerfile build order to speedup iterations
In order to get faster iterations, the Dockerfile should be altered in a way that Docker can cache most of its layers during rebuilds. Static layers should be moved up and leave down the one that changes more often.

I will submit a PR with the proposed changes.

### PR DESCRIPTION
**Pull request by gbisheimer**
Fixes #427

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,4 +12,6 @@ RUN ln -s /portus/docker/crono-supervisord.conf /etc/supervisor/conf.d/crono.con
 
 ENV COMPOSE=1
 
+ADD . /portus
+
 EXPOSE 3000
```
Comment (flavio) [inline Dockerfile]: This can be moved up

File: Dockerfile [hunk 2]
```diff
@@ -12,4 +12,6 @@ RUN ln -s /portus/docker/crono-supervisord.conf /etc/supervisor/conf.d/crono.con
 
 ENV COMPOSE=1
 
+ADD . /portus
```
Comment (flavio) [inline Dockerfile]: Right now the docker-compose setup is for development purposes: the git checkout is always mounted into the running container. Hence we can even drop this directive.

Comment (gbisheimer) [inline Dockerfile]: That's correct for development, but I'm planning to use docker containers to deploy to production as well. So I need to have the code inside the container.
The only thing that differs is the volume mount and the environment variables (RAILS_ENV)

**Comment by flavio** [comment]
Ok, just move the `EXPOSE` directive upper. Then I'll merge the PR

**Comment by flavio** [comment]
Superseded by #524 524

**Pull request closed without merging**